src/CONST.js

```diff
@@ -490,8 +490,8 @@ const CONST = {
         WIDTH: 320,
         HEIGHT: 400,
     },
-    NON_NATIVE_EMOJI_PICKER_LIST_HEIGHT: 300,
-    EMOJI_PICKER_ITEM_HEIGHT: 40,
+    NON_NATIVE_EMOJI_PICKER_LIST_HEIGHT: 295,
+    EMOJI_PICKER_ITEM_HEIGHT: 35,
```

Comment:
Why do we need to change this? Let's try to keep 40 as that's the standard button size we use.

Reply:
Actual Height of emoji is 34.5 (After setting lineHeight to 27, it will be 35). So, changed EMOJI_PICKER_ITEM_HEIGHT to match value in UI.
<img width="459" alt="Screenshot 2022-11-02 at 15 03 49" src="https://user-images.githubusercontent.com/25876548/199453752-6285ee8d-3dfe-4781-88e4-0945f92de6ec.png">

Reply:
> Could you please confirm whether the design of the emoji list is looking fine in the EmojiPicker? we changed line-Height and height of the emoji.

Slight change is UI is due to the use of lineHeight: 27 instead of 26.5 [here](https://github.com/Expensify/App/pull/8902/files#diff-4c815c48cfd171056c4be0e2bc556302e115c20c2bef5b5b00dbe55450faca5aR40). 

cc: @parasharrajat

Reply:
Can you post a comparison picture of Staging vs this branch?

Reply:
We should not approach this about trying to "match value in UI" - we very specifically want these buttons to be 40px tall as that is our standard button size.

Reply:
I can set emojiHeight to 40. Do we want to change lineHeight as well? It is 27 now.

https://user-images.githubusercontent.com/25876548/199550363-73f355cc-5db6-4caa-9627-043aed9956d4.mov

Reply:
Changing the line height to 40 makes sense to me so that the emoji appears vertically centered in the button. Or you could just unset the line height and use something like 1, as long as the emoji gets vertically centered within that area.

Reply:
> Also, I found that some emojis has a different height. So, we need to add lineheight in emojiText in styles.js

https://github.com/Expensify/App/pull/8902#issuecomment-1133906314 Line-height was added for the above reason. you will have to either adjust the line-height or use a flexbox to center the emoji on 40px. 

I am curious what was the height before this PR.

Reply:
This is how it looks on staging. I can confirm that it does not look like 40px height.
![image](https://user-images.githubusercontent.com/24370807/199554089-71f4cbc0-877f-42f4-9d83-ab775c9252ee.png)

Reply:
> This is how it looks on staging. I can confirm that it does not look like 40px height.

<img width="459" alt="Screenshot 2022-11-02 at 15 03 49" src="https://user-images.githubusercontent.com/25876548/199453752-6285ee8d-3dfe-4781-88e4-0945f92de6ec.png">

Reply:
| Before | after|
|--------|-------|
|![image](https://user-images.githubusercontent.com/24370807/199555234-e3707434-a69d-4b54-a64c-43bd10f50b94.png) | ![image](https://user-images.githubusercontent.com/24370807/199555187-f8d0382e-9e30-4fdc-89fc-9f08a627ab2d.png)|

What do you say @shawnborton ?

Reply:
Hmm yeah, maybe that is too tall. How about we just use 32px then for everything? This way it's still based off of the 4px unit.

Reply:
| Before             |  After (with 32px) |
:-------------------------:|:-------------------------:
<img width="324" alt="Screenshot 2022-11-02 at 23 08 34" src="https://user-images.githubusercontent.com/25876548/199559990-7c4ef700-555d-4564-8fd3-7d33f507608d.png"> | <img width="324" alt="Screenshot 2022-11-02 at 23 08 59" src="https://user-images.githubusercontent.com/25876548/199560065-c819c34c-199c-47d8-b8e5-66b5ba92ae6c.png">

Reply:
| Before             |  After (with 36px) |
:-------------------------:|:-------------------------:
<img width="324" alt="Screenshot 2022-11-02 at 23 08 34" src="https://user-images.githubusercontent.com/25876548/199559990-7c4ef700-555d-4564-8fd3-7d33f507608d.png"> | <img width="324" alt="Screenshot 2022-11-02 at 23 17 24" src="https://user-images.githubusercontent.com/25876548/199560863-97de6d65-0b59-4dd3-973f-e93502cc3eea.png">

Reply:
| Before             |  After (with 40px) |
:-------------------------:|:-------------------------:
<img width="324" alt="Screenshot 2022-11-02 at 23 08 34" src="https://user-images.githubusercontent.com/25876548/199559990-7c4ef700-555d-4564-8fd3-7d33f507608d.png"> | <img width="324" alt="Screenshot 2022-11-02 at 23 20 52" src="https://user-images.githubusercontent.com/25876548/199561510-ab831767-c857-42f3-8dc4-9122a9de0c07.png">

Reply:
@shawnborton @parasharrajat Can you please confirm which value should I use?

Reply:
I like 32px, let's go with that!

Reply:
Yeah, 32px is good. Let's do that.

Reply:
Update pushed.